src/functions/negate.js

```diff
@@ -3,13 +3,39 @@ import {wrap} from "../core/wrap";
 // Get the complement of a function.
 export const negate = wrap({
     name: "negate",
+    summary: "Get the logical negation of a predicate function.",
```

Comment:
So I thought about this all day and I think you should leave  it in the production builds. It won't increase build size all that much, and it could provide a lot of value to people who are new to these kinds of programming concepts. Not everyone is going to know what `lexOrder` or `ngrams` mean. Instead of having to Google around, it'd be really handy if a short summary was included with the method itself. Very neat idea.